docs/changes/cosmology/12354.api.rst

```diff
@@ -0,0 +1 @@
+`Planck18_arXiv_v2` is removed.
```

Comment:
Maybe be a bit more explicit: "The already deprecated ... has been removed. Use ... instead."